devservices/programs.conf

```diff
@@ -0,0 +1,3 @@
+[program:devserver]
+command=cargo run
```

Comment:
If the intention is to have Relay running in a way where it can run as a Relay docker replacement for Sentry ingestion, we need different flags here, just to have a Relay running as a managed instance, we still need more cli flags, but the cargo command is fine.

Reply:
These flags can still be passed in - `devservices serve --all-features -- config init` or `devservices serve --all-features -- run